eng/Subsets.props

```diff
@@ -394,7 +394,7 @@
       Test="true" Category="tools"/>
   </ItemGroup>
 
-  <ItemGroup Condition="$(_subset.Contains('+clr.nativecorelib+'))">
+  <ItemGroup Condition="$(_subset.Contains('+clr.nativecorelib+')) and '$(TargetArchitecture)' != 'arm'">
```

Comment:
A temporary workaround for the crossgen failure. Maybe there is a better solution...